ui/src/app/SiteModule/context/test.js

```diff
@@ -11,9 +11,9 @@ export const getSiteData = async () => {
 
 export const getMenus = async () => {
   const data = [
-    { name: "primary", items: [{ text: "test", link: "/test" }] },
-    { name: "secondary", items: [{ text: "test", link: "/test" }] },
-    { name: "footer", items: [{ text: "test", link: "/test" }] },
+    { slug: "primary", items: [{ text: "test", url: "/test" }] },
+    { slug: "secondary", items: [{ text: "test", url: "/test" }] },
+    { slug: "footer", items: [{ text: "test", url: "/test" }] },
   ];
   return data;
 };
```

Comment:
it'd be cool if we could come up with a way to inject separate contexts at the entrypoint, rather than conditionally import them at this depth

Reply:
That's the benefit of injection vs import?